samples/OmniKassa.Samples.DotNet461/Controllers/HomeController.cs

```diff
@@ -32,17 +32,35 @@ public class HomeController : Controller
 
         public HomeController()
         {
+            var appSettings = ConfigurationManager.AppSettings;
+            SIGNING_KEY = appSettings["SigningKey"];
+            TOKEN = appSettings["RefreshToken"];
+            RETURN_URL = appSettings["CallbackUrl"];
+            BASE_URL = appSettings["BaseUrl"];
+            
```

Comment:
Only thing that might be handy here is a check that if BASE_URL = SANDBOX or PRODUCTION the BASE_URL is set to the url in the environment or else it will stay how it is :-) So customers do not have to fill in the sandbox or production url

Reply:
implemented with latest commit: https://github.com/rabobank-nederland/omnikassa-dotnet-sdk/pull/21/commits/d517c9034cb5d10302cf1c0d6ccf289d32c2c644